docs/src/pages/[platform]/index.page.tsx

```diff
@@ -46,7 +46,7 @@ const HomePage = ({ colorMode }) => {
     return () => {
       document.removeEventListener('scroll', handleScroll);
     };
-  }, []);
+  }, [handleScroll]);
```

Comment:
Is it intentional to run this effect for every render of `HomePage`?

Reply:
fixed